ui/common/css/theme/board/_board-2d.scss

```diff
@@ -0,0 +1,41 @@
+@import 'boards';
+
+@each $name, $board-theme in $board-themes-2d {
+  body[data-board='#{$name}'] .is2d {
+    cg-board {
+      $file-ext: map-get($board-theme, file-ext);
+      $name-override: map-get($board-theme, name-override);
+      $file-name: if($name-override, $name-override, $name);
+      $dir-name: 'board#{if($file-ext == "svg", "/svg", "")}';
+      &::before {
```

Comment:
all boards, main and mini, have lost their rounded borders.

Reply:
ah, yes. it should be back now, the box shadows too.